pkgs/by-name/ri/rime-japanese/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "rime-japanese";
+  version = "0-unstable-2023-08-02";
+
+  src = fetchFromGitHub {
+    owner = "gkovacs";
+    repo = "rime-japanese";
+    rev = "4c1e65135459175136f380e90ba52acb40fdfb2d";
+    hash = "sha256-/mIIyCu8V95ArKo/vIS3qAiD8InUmk8fAF/wejxRxGw=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/rime-data/
+    cp japanese.*.yaml $out/share/rime-data/
```

Just a suggestion, since I'm already posting. Not a blocker :)

```suggestion
    install -D japanese.*.yaml -t "$out/share/rime-data"
```